snapcraft/plugins/ant.py

```diff
@@ -83,15 +192,44 @@ def build(self):
         for prop, value in self.options.ant_properties.items():
             command.extend(["-D{}={}".format(prop, value)])
 
-        self.run(command)
-        files = glob.glob(os.path.join(self.builddir, "target", "*.jar"))
+        self.run(command, rootdir=self.builddir)
+        files = glob(os.path.join(self.builddir, "target", "*.jar"))
         if files:
             jardir = os.path.join(self.installdir, "jar")
             os.makedirs(jardir)
             for f in files:
                 base = os.path.basename(f)
                 os.link(f, os.path.join(jardir, base))
 
+        self._create_symlinks()
+
+    def _create_symlinks(self):
+        if self.project.info.base not in ("core18", "core19"):
```

Comment:
This is in a couple of places. Would it be better to just put this once in `__init__`?

Reply:
To make porting easier, image you now come in with a fedora base.

Reply:
That's actually exactly what I'm imagining: adding "fedora" to one of those tuples and forgetting the other one. Can we at least extract this to a method or some other way to only need to modify the list of supported bases once?

Reply:
ok, but this goes into the TODO bucket.